### PR TITLE
Create new aliases and edited comingsoon

### PR DESCRIPTION
**Pull request by yehudahrrs**
Alias for openthefloodgates
Also a few aliases for !tag anticheat
edited comingsoon tag

### CODE REVIEW

**Review by Heath123 — APPROVED** [review]

File: tags/outdated.tag [hunk 1]
```diff
@@ -1,5 +1,2 @@
-type: text
-
----
-
-Please read this when considering what Minecraft version to use for your server: https://matthewmiller.dev/blog/which-minecraft-version/
\ No newline at end of file
+type: alias
+target: outdatedclient
```
Comment (rtm516) [inline tags/outdated.tag:2]: This no longer works as it used to, this needs to be looked at more.

**Review by rtm516 — APPROVED** [review]

**Comment by rtm516** [comment]
We need to come up with a sub-folder system for tags otherwise the directory will be littered with aliases.

File: tags/outdated.tag (hunk 1 above)
Comment (yehudahrrs) [reply tags/outdated.tag:2]: ?

**Comment by yehudahrrs** [comment]
> We need to come up with a sub-folder system for tags otherwise the directory will be littered with aliases.

ahh, ok

**Pull request closed without merging**